tmt/cli.py

```diff
@@ -1752,6 +1752,9 @@ def init(
 @option(
     "--epel", is_flag=True, default=False,
     help="Enable epel repository.")
+@option(
+    "--install", default=[], metavar="PACKAGE", multiple=True,
+    help="Install local rpm package on the guest.")
```

Comment:
It's not `local` packages only, right? We can install packages from repositories as well. Plus, the `install` plugin supports non-rpm packages as well.

Reply:
Fixed in 1b7d1f59.